.gitlab/maintenance_jobs/kitchen.yml

```diff
@@ -38,8 +38,6 @@ periodic_kitchen_cleanup_azure:
     - RESOURCE_GROUP_PREFIX=kitchen-win python3 /deploy_scripts/cleanup_azure.py
     - RESOURCE_GROUP_PREFIX=kitchen-security-agent python3 /deploy_scripts/cleanup_azure.py
     - RESOURCE_GROUP_PREFIX=kitchen-system-probe python3 /deploy_scripts/cleanup_azure.py
```

Comment:
```suggestion
```
I think we can also delete this one, since https://github.com/DataDog/datadog-agent/pull/24755/files#diff-84678774d5b07132fa69f4622b9efe596dfa24153cf7bc51f58b9ca15fad7645 was merged

Reply:
And same for `install-script` `upgrade` and `step-by-step` prefixes. It was forgotten when those tests were migrated